boa_unicode/build_tables.js

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env node
```

Comment:
I would add a section in the `CONTRIBUTING.ml` file to explain how to use this script, what dependencies it has and how to run it locally to update the tables. Also, a short explanation on what it does and why we need it would be nice. Maybe some comments in the script itself.

Reply:
I prefer to add file headers in `build_table.js` or add a `README.md` under `boa_unicode` to keep the top-level `CONTRIBUTING.md` clean. Is one of them good for you?

Reply:
Let's create a `README.md` file under `boa_unicode` then. But I would at least add a link to the `CONTRIBUTING.md` file, to make sure everything is linked, this being part of the project too.

Reply:
Will do :)